crowbar_framework/app/assets/javascripts/application.js

```diff
@@ -104,7 +104,7 @@ jQuery(document).ready(function($) {
   });
 
   setBlockUI('submit', $('[data-blockui]'));
-  setBlockUI('click', $('body[class="installer/upgrades"] [data-blockui]'));
+  setBlockUI('click', $('body[class="installer/upgrades"]').find('[data-blockui]'));
```

Comment:
If you use proper selectors it should work. Without testing it I'm pretty sure `body[class="installer/upgrades"] > [data-blockui]` should work.